<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIncome</class>
 <widget class="QDialog" name="AddIncome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sitka Banner</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AddIncome</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/ghjk.png</normaloff>:/images/ghjk.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add Incomes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>69</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Amount">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Income Source</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="IncomeSource">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Job</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Allowance from parents</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scholarships</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Student aid</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="DateIncome">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="IncomeDesc">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="IncomeTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>551</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>231</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="SaveIncome">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetIncome">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-230</x>
     <y>-10</y>
     <width>1291</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/6070859.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BackInc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/left-arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>IncomeTable</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>BackInc</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
